www/js/control/DataDatePicker.tsx

```diff
@@ -0,0 +1,48 @@
+// this date picker element is set up to handle the "download data from day" in ProfileSettings
+// it relies on an angular service (Control Helper) but when we migrate that we might want to download a range instead of single
+
+import React from "react";
+import { DatePickerModal } from 'react-native-paper-dates';
+import { useTranslation } from "react-i18next";
+import { getAngularService } from "../angular-react-helper";
+
+const DataDatePicker = ({date, setDate, open, setOpen}) => {
+  const { t } = useTranslation();
+  const ControlHelper = getAngularService("ControlHelper");
+  const { i18n } = useTranslation('general-settings.choose-date'); //able to pull lang from this
```

Comment:
`useTranslation()` does not need this argument

Reply:
I'm not sure why I thought I needed that, I just took it out